fuse/plugins/micro_physics/lineage_cluster.py

```diff
@@ -362,7 +409,6 @@ def classify_lineage(particle_interaction):
         return NEST_NONE
 
 
-@njit()
```

Comment:
Did you check how the speed of the code changes when removing numba? (Just want to make sure we do not slow done too much)

Reply:
Not checked yet.. but on small runs I saw no notable changes.